src/tools/bin/nextstep_from_period.sh

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env bash
+#
+# Copyright 2017-2018 Axel Huebl, Ilja Goethel
+#
+# This file is part of PIConGPU.
+#
+# PIConGPU is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PIConGPU is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PIConGPU.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
+set -euf -o pipefail
+
+helpText()
+{
+    echo "
+    Usage: nextstep_from_period.sh period lastStep [currentStep]
+
+    Script returns next timestep specified by the period syntax
+    Takes three positional arguments:
+     - a string representing period-syntax, like 3000,12000:15000:1000,1337:1337
+     - the current timestep (convertible to int)
```

Comment:
order is different than above here

Reply:
ups, yeah, it's wrong; I reordered it to make the last argument optional